students/11th/hanteagyu/main.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>westagram main page</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+      <div class="full">
+          <header>
+              <nav>
+                  <div class="navBox">
+                      <div class="navSize">
+                          <div class="logoBox">
+                              <a class="logo" href="?"><img class="logo" src="img/logo_text.png"></a>
+                          </div>
+                          <div class="search">
+                              <input class="searchBox" type="text" placeholder="검색">
+                          </div>
+                          <div class="navLinkBox">
+                              <div class="navLinkFrame">
+                                  <a href="?"><img class="navLink" src="img/home.png"></a>
+                                  <a href="?"><img class="navLink" src="img/plane.png"></a>
+                                  <a href="?"><img class="navLink" src="img/compass.png"></a>
+                                  <a href="?"><img class="navLink" src="img/heart.png"></a>
+                                  <img  class="navLink" id= "roundBorder" src="img/empty%20face.png">
```

Comment:
```suggestion
                                  <img class="navLink" id="roundBorder" src="img/empty%20face.png">
```
1. 속성과 값 사이 공백은 없애주세요
2. 가능하면 id 대신 class selector 사용해주세요